week_5/char-yb/week5/README.md

```
@@ -0,0 +1,13 @@
+## 5주차
+
+- todo-list 구현해보기, CRUD
+  - todo 생성하기
+  - todo 단건 조회
+  - todo 상태로 조회 (예정, 진행중, 완료), 상태는 index 없이.
+    - equals 비교
+ - test case를 만드는 api 구성. 5000만건 Todo 생성 후 진행
```

Markdownlint (0.35.0)

Expected: 0; Actual: 1
Inconsistent indentation for list items at the same level

(MD005, list-indent)